docs/server-api/admin-apis/users.md

```diff
@@ -41,6 +41,8 @@ Except for username and password, all other fields are optional.
 | zip        | string     | Pincode of the user                                                                 |
 | state      | string     | State of user in ISO 3166-2:IN format (https://en.wikipedia.org/wiki/ISO_3166-2:IN) |
 | phone      | string     | Phone of the user                                                                   |
+| batches    | list       | Batches of the user                                                                 |
+| expires    | datestring | Expire date of the batches added                                                    |
```

Comment:
Expir**y** date of the batches subscription. Format yyyy-mm-dd. After which the user will be removed form the batch.